uploadNeuroDB/tarchiveLoader

```diff
@@ -471,9 +471,10 @@ if ($verbose){
 ################################################################
 # If no good data was found stop processing and write error log.
 ################################################################
-if ($mcount < 1) { 
+if ($mcount < 1) {
+    my $excluded_series = join(',', map { quotemeta($_) } @$exclude);
     $message = "\nNo data could be converted into valid MINC files. ".
-               "Localizers will not be considered! \n" ; 
+               "$excluded_series will not be considered! \n" ;
```

Comment:
I think it is possible that `@$exclude` does not contain any elements. The second part of the error message should be displayed only if `@$exclude` is not empty, as in:

```perl
if ($mcount < 1) {
    $message = "\nNo data could be converted into valid MINC files. ";
    if (@$exclude) {
        my $excluded_series = join(',', map { quotemeta($_) } @$exclude);
        $message .= "$excluded_series will not be considered! \n" ;
    }
    .
    .
    .
```

Reply:
@nicolasbrossard done! Thank you!!